src/main/java/TestFile2.java

```diff
@@ -0,0 +1,16 @@
+class TestFile2 {
+    public String foo() {
+        return "bar";
+    }
+
+    private static void incorrectAssignmentInIfCondition() {
```

Comment:
[Checkstyle] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2.